resdk/tests/functional/permissions/e2e_permissions.py

```diff
@@ -6,50 +6,54 @@
+
+    def tearDown(self):
+        super().tearDown()
 
+        self.test_collection.delete(force=True)
+
+    def test_permissions(self):
         # User doesn't have the permission to view the collection.
         with self.assertRaises(LookupError):
-            self.user_res.collection.get(collection_admin.id)
+            self.user_res.collection.get(self.test_collection.id)
 
-        collection_admin.permissions.add_user(USER_USERNAME, 'view')
+        self.test_collection.permissions.add_user(USER_USERNAME, 'view')
 
         # User can see the collection, but cannot edit it.
-        collection_user = self.user_res.collection.get(collection_admin.id)
-        collection_user.name = 'New test collection'
+        user_collection = self.user_res.collection.get(self.test_collection.id)
         with self.assertRaises(ResolweServerError):
-            collection_user.save()
+            user_collection.name = 'Different name'
+            user_collection.save()
 
-        collection_admin.permissions.add_user(USER_USERNAME, 'edit')
+        self.test_collection.permissions.add_user(USER_USERNAME, 'edit')
 
         # User can edit the collection.
-        collection_user = self.user_res.collection.get(collection_admin.id)
-        collection_user.name = 'New test collection'
-        collection_user.save()
+        user_collection.name = 'Different name'
+        user_collection.save()
 
-        collection_admin.permissions.remove_user(USER_USERNAME, 'edit')
+        self.test_collection.permissions.remove_user(USER_USERNAME, 'edit')
 
         # Edit permission is removed again.
-        collection_user = self.user_res.collection.get(collection_admin.id)
-        collection_user.name = 'Another collection'
         with self.assertRaises(ResolweServerError):
-            collection_user.save()
+            user_collection.name = 'Different name 2'
+            user_collection.save()
 
     def test_get_holders_with_perm(self):
-        collection = self.res.collection.create(name='Test collection')
-        collection.permissions.add_user(USER_USERNAME, ['edit', 'view'])
-        collection.permissions.add_public('view')
-
-        self.assertEqual(len(collection.permissions.owners), 1)
-        self.assertEqual(collection.permissions.owners[0].get_name(), "admin")
+        self.test_collection.permissions.add_user(USER_USERNAME, ['edit', 'view'])
+        self.test_collection.permissions.add_public('view')
 
-        self.assertEqual(len(collection.permissions.editors), 2)
-        self.assertEqual(collection.permissions.editors[0].get_name(), "admin")
-        self.assertEqual(collection.permissions.editors[1].get_name(), "user")
+        self.assertEqual(len(self.test_collection.permissions.owners), 1)
+        self.assertEqual(self.test_collection.permissions.owners[0].get_name(), "admin")
 
+        self.assertEqual(len(self.test_collection.permissions.editors), 2)
```

Comment:
These lines are a part of the previous commit.

Reply:
They were created in previous commit, but were edited as part of this comit.